config/shapes/channel_selection.py

```diff
@@ -1,26 +1,116 @@
 from ntuple_processor.utils import Selection
 
 
-def channel_selection(channel, era, special=None):
+def channel_selection(channel, era, wp, vs_ele, special=None):
     # Specify general channel and era independent cuts.
     cuts = [
         ("extraelec_veto<0.5", "extraelec_veto"),
         ("extramuon_veto<0.5", "extramuon_veto"),
         ("dimuon_veto<0.5", "dilepton_veto"),
         ("q_1*q_2<0", "os"),
     ]
+
+    vs_ele_dict = {
+
+        "vvtight" : "VVTight",
+        "vtight" : "VVTight",
+        "tight" : "Tight",
+        "medium" : "Medium",
+        "loose" : "Loose",
+        "vloose" : "VLoose",
+        "vvloose" : "VVLoose",
+        "vvvloose" : "VVVLoose",
+    }
+
+    if vs_ele not in vs_ele_dict.keys():
+        print("This working point doen't exist. Please specify the correct vsEle discriminator ")
+    vs_ele_discr = vs_ele_dict[vs_ele]
+
     if special is None:
         if "mt" in channel:
             #  Add channel specific cuts to the list of cuts.
-            cuts.extend(
-                [
-                    ("id_tau_vsMu_Tight_2>0.5", "againstMuonDiscriminator"),
-                    ("id_tau_vsEle_VVLoose_2>0.5", "againstElectronDiscriminator"),
-                    ("id_tau_vsJet_Tight_2>0.5", "tau_iso"),
-                    ("iso_1<0.15", "muon_iso"),
-                    ("mt_1 < 70", "mt_cut"),
-                ]
-            )
+            if wp == "vvtight":
+                cuts.extend(
+                    [
```

Comment:
cant we put this if statements into a single function as well (using the wp dict on top with something like 

```py

(f"id_tau_vsJet_{vs_ele_dict[wp]}_2>0.5", "tau_iso"),
```